complete/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css" />
+  <script src="script.js"></script>
+  <meta name="description" content="Responsive Website">
+  <title>Responsive Website</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/meyer-reset/2.0/reset.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  </head>
+  <body>
+    <header class="header">
+    <div class="top-container">
+    <div class="logo"><img src="images/Logo.png" alt="logo"/></div>
+    <nav class="top-navbar">
+    <ul>
+      <li><a class="active" href="#home">Home</a></li>
+      <li><a href="#pages">Pages</a></li>
+      <li><a href="#features">Features</a></li>
+      <li><a href="#extensions">Extensions</a></li>
+      <li><a href="#tutorials">Tutorials</a></li>
+      <li><a href="#contact_us">Contact us</a></li>
+      </ul>
+      </nav>
+      <div class="top-nav-social">
+      <a href="#" class="fa fa-facebook"></a>
+      <a href="#" class="fa fa-twitter"></a>

```

Comment:
![social](https://user-images.githubusercontent.com/2418147/42779073-8a9b0e32-893f-11e8-8637-5538db8f1f4c.jpg)

Can we make the icons have a subtle highlight when hovering with the mouse? see picture with hover on the twitter icon